### PR TITLE
Add elapse logs in compaction

### PR DESCRIPTION
**Pull request by XuanYang-cn**
See also: #15413

Signed-off-by: yangxuan <xuan.yang@zilliz.com>

/kind improvement

### CODE REVIEW

File: internal/datanode/compactor.go [hunk 1]
```diff
@@ -174,12 +174,14 @@ func (t *compactionTask) mergeDeltalogs(dBlobs map[UniqueID][]*Blob, timetravelT
 
 	dbuff.updateSize(dbuff.delData.RowCount)
 	log.Debug("mergeDeltalogs end", zap.Int64("PlanID", t.getPlanID()),
-		zap.Int("number of pks to compact in insert logs", len(pk2ts)))
+		zap.Int("number of pks to compact in insert logs", len(pk2ts)),
+		zap.Any("elapse", time.Since(mergeStart)))
```
Comment (sunby) [inline internal/datanode/compactor.go:178]: please transfer elapse to millisecond unit

Comment (sunby) [inline internal/datanode/compactor.go:178]: please transfer elapse to millisecond unit

Comment (sunby) [inline internal/datanode/compactor.go:178]: please transfer elapse to millisecond unit

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**